docs/source/reference/storages.rst

```diff
@@ -27,17 +27,17 @@ optuna.storages.journal
    :toctree: generated/
    :nosignatures:
 
-   optuna.storages.journal.JournalFileBackend
-   optuna.storages.journal.JournalRedisBackend
+   journal.JournalFileBackend
+   journal.JournalRedisBackend
 
 Users can flexibly choose a lock object for :class:`~optuna.storages.journal.JournalFileBackend` and here is the list of supported lock objects:
 
 .. autosummary::
    :toctree: generated/
    :nosignatures:
 
-   optuna.storages.journal.JournalFileSymlinkLock
-   optuna.storages.journal.JournalFileOpenLock
+   journal.JournalFileSymlinkLock
+   journal.JournalFileOpenLock
```

Comment:
I think it would be better to remove the prefix of `journal.` here. It is obvious that these are placed inside `optuna.storages.journal`.

Reply:
I'm unsure why these changes were not suggested in sphinx's build message https://github.com/optuna/optuna/actions/runs/11640162393/job/32417223006. Anyway, I'll take a look at them. Thanks.

Reply:
When I removed `journal.` from the lines, the sphinx shows the following warning messages:
```
WARNING: [autosummary] failed to import optuna.storages.JournalFileBackend.
Possible hints:
* ModuleNotFoundError: No module named 'optuna.storages.JournalFileBackend'
* ImportError:
* AttributeError: module 'optuna.storages' has no attribute 'JournalFileBackend'
WARNING: [autosummary] failed to import optuna.storages.JournalRedisBackend.
Possible hints:
* ImportError:
* ModuleNotFoundError: No module named 'optuna.storages.JournalRedisBackend'
* AttributeError: module 'optuna.storages' has no attribute 'JournalRedisBackend'
```

Reply:
Maybe alias causes this issue. in my understanding, `DeprecatedJournalFileOpenLock` and `DeprecatedJournalFileSymlinkLock` are actual class names.

Reply:
Let me clarify this a little bit more.
To remove `journal` prefix here, the definition, they should be inhttps://github.com/optuna/optuna/blob/master/optuna/storages/__init__.py. However, 
`optuna.storages.JournalFileSymlinkLock` and `JournalFileOpenLock` are aliases of `DeprecatedJournalFileSymlinkLock` and `DeprecatedJournalFileOpenLock`, respectively.

Reply:
In my mind importing the following classes 

```python
JournalFileBackend
JournalRedisBackend
JournalFileSymlinkLock  # deprecatedのaliasではない方
JournalFileOpenLock  # deprecatedのaliasではない方
``` 
in `optuna/storages/__init__.py` is an approach to remove the prefix.

Reply:
Thanks for the investigation. According to your survey, I wouldn't like to add deprecated class into `__init__.py`. So, current direction is fine. Let me resolve this conversation. Thank you so much.